CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -459,8 +459,8 @@
         <note/>
       </trans-unit>
       <trans-unit id="Known For" xml:space="preserve">
-        <source>Known For</source><target>Bekannt aus</target>
-        <note/>
+        <source>Known For</source><target state="needs-review-translation">Bekannt für</target>
+        <note/><note from="reviewer">The "known for" roles are not translated; it still says "Acting", "Camera", "Directing"</note>
```

Comment:
I guess these come from the API? Will you be able to isolate them somehow?

![IMG_7321](https://github.com/cliss/callsheet-localizations/assets/18497564/556c96fe-0b5b-4b7e-b7fe-cc9323dad565)

Reply:
That’s right; I don’t think there’s much I can do about it, unfortunately.

Reply:
Is it a finite amount of values you get from them? If you had a list of translations for the value for all key-value-pairs, would it slow everything down / complicate everything immensely if you parsed the API call reply differently (with value substitution) depending on the language of the UI?

Reply:
I think there is a set of possible values, but I don't know of any way to query TMDB for that set. :(

Reply:
There would be the option (in case this is not too much work) to collect as many key:value pairs as we can, translate what we found, and just pass through without translation those we didn't catch, as a fallback.

I have absolutely no idea how much programming work this would be though...